proguard/proguard-moshi.pro

```diff
@@ -0,0 +1,2 @@
+-keep class com.squareup.moshi.JsonAdapter
+-keep class com.squareup.moshi.Moshi
```

Comment:
moshi needs to follow proguard rule according by official https://github.com/square/moshi#proguard

issue link: https://github.com/square/moshi/pull/342